Formula/i2pd.rb

```diff
@@ -0,0 +1,64 @@
+class I2pd < Formula
+  desc "Full-featured C++ implementation of I2P client"
+  homepage "https://i2pd.website"
+  url "https://github.com/PurpleI2P/i2pd/archive/2.19.0.tar.gz"
+  sha256 "7202497ffc3db632d0f7fed93eafaf39aa75efea199705dae7d022249b069eb9"
+
+  depends_on "make" => :build
+  depends_on "boost"
+  depends_on "libressl"
+  depends_on "miniupnpc"
+
+  needs :cxx11
+
+  def install
+    ENV["HOMEBREW_OPTFLAGS"] = "-march=#{Hardware.oldest_cpu}" unless build.bottle?
+
+    system "make", "HOMEBREW=1", "USE_UPNP=yes"
+    bin.install "i2pd"
```

Comment:
Is there a `make install` target we can use instead? If not, please file an issue upstream requesting one so that we can avoid the manual `bin.install`, `man1.install`, etc.

Reply:
We not using `install` target in make, but I can add it if that prefered.